tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1.4
 ###### Minimal image with base system requirements for most stages
-FROM docker.io/ubuntu:22.04 AS minimal
+FROM docker.io/ubuntu:25.04 AS minimal
```

Comment:
We should not upgrade ubuntu yet, as this version was pinned based on upstream course-discovery.